libraries/standard/http/cbmc/proofs/HTTPClient_Send/HTTPClient_Send_harness.c

```diff
@@ -53,28 +53,20 @@ void harness()
     __CPROVER_assume( isValidHttpResponse( pResponse ) );
 
     /* Initialize transport interface. */
-    ( void ) allocateTransportInterface( &transportInterface );
+    pTransportInterface = allocateTransportInterface( NULL );
 
-    if( nondet_bool() )
+    if( pTransportInterface != NULL )
```

Comment:
Let's confirm that this doesn't kill performance.  What you have is logically correct.  But the nondet_bool() was part of the speedup.  Something similar to this generated 300M clauses and we got it down to 280K clauses.  This is important.

Reply:
It does not I have tried it locally and it completes within 5 minutes.

Reply:
```
Solving with MiniSAT 2.2.1 with simplifier
526485 variables, 2138103 clauses
```